_mappings/supported-field-types/index.md

```diff
@@ -125,6 +125,9 @@ PUT testindex1/_doc/2
 }
 ```
 
+semantic field cannot support an array of values since we will map it to an embedding field as rank_features/knn_vector, and they cannot support multiple vectors.
```

Comment:
```suggestion
The `semantic` field cannot contain an array of values because it's mapped to an embedding field (`rank_features` or `knn_vector`), which supports only a single vector.
```